CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Change Log for Tensile
 
+## (Unreleased) Tensile 4.35.0
+### Added
+- Async DMA support for Transpose Data Layout (ThreadSeparateGlobalReadA/B)
+- Option to output library logic in dictionary format
+- No solution found error message for benchmarking client
+- Exact K check for StoreCInUnrollExact
+- Support for CGEMM + MIArchVgpr
+- client-path parameter for using prebuilt client
+- CleanUpBuildFiles global parameter
+- Debug flag for printing library logic index of winning solution
+- NumWarmups global parameter for benchmarking
+- Windows support for benchmarking client
+- DirectToVgpr support for CGEMM
+- TensileLibLogicToYaml for creating tuning configs from library logic solutions
+### Optimizations
+- No syncThreads if number of threads in block equal to wavefront size
```

Comment:
I think this is reverted.

Reply:
> I think this is reverted.

Precisely, commented out.

Reply:
Thanks for your update.
I do not have any further comment.
Waiting for others.